chia/consensus/blockchain.py

```diff
@@ -580,7 +554,7 @@ async def validate_unfinished_block(
             not self.contains_block(block.prev_header_hash)
             and not block.prev_header_hash == self.constants.GENESIS_CHALLENGE
         ):
-            return PreValidationResult(uint16(Err.INVALID_PREV_BLOCK_HASH.value), None, None)
+            return PreValidationResult(uint16(Err.INVALID_PREV_BLOCK_HASH.value), None, None, False)
```

Comment:
it's hard to know what this `False` means here, without a named parameter or a named flag/enum.

Reply:
Not sure if we can force named parameters in streamable

Reply:
I see, this is a `dataclass` thing.